src/components/settings/settings.tsx

```diff
@@ -277,6 +282,19 @@ class Settings extends Component<any, any> {
     return activeVideoTrackHeight ? getLabelBadgeType(activeVideoTrackHeight) : null;
   }
 
+  getQualityLabel(buttonBadgeType): string {
+    switch (buttonBadgeType) {
```

Comment:
I added this strings in the en.json, also added default value because currently we don't have the translation for all the languages.
This request coming from customer and product approved this.